R/circa_single.R

```diff
@@ -11,6 +11,7 @@
 #' @param timeout_n The upper limit for the model fitting attempts. Default is 10,000.
 #' @param return_figure Whether or not to return a ggplot graph of the rhythm and cosine model.
 #' @param control \code{list}. Used to control the parameterization of the model.
+#' @param sample_weights A numeric vector of per-sample weights to allow downweighting of outlier samples.
```

Comment:
Could we please change the argument name to be `weights` to keep it consistent with `nls()` which is being called. I'd also prefer if the argument description was copied from `?nls()`
> `weights` an optional numeric vector of (fixed) weights. When present, the objective function is weighted least squares.

Reply:
Renamed `sample_weights` to `weights` everwhere.